tools/bigquery-zos-mainframe-connector/gszutil/src/main/scala/com/google/cloud/bqsh/Bqsh.scala

```diff
@@ -184,8 +217,10 @@ object Bqsh extends Logging {
         if (!(singleQuoted || doubleQuoted)) {
           if (c == '\'') {
             singleQuoted = true
+            sb.append(c)
```

Comment:
The test [it should "replace variables" in](https://github.com/jasonmar/professional-services/blob/mainframe-connector-a/tools/bigquery-zos-mainframe-connector/gszutil/src/test/scala/com/google/cloud/bqsh/ShellSpec.scala#L99) Seems to have been broken by this change. 

Actually, If my interpretation is correct, I would say the test should be modified to reflect the current behavior of this function since `replaceEnvVars` no longer removes quotes.

Reply:
updated the test